src/qibocal/protocols/characterization/coherence/t1.py

```diff
@@ -7,6 +7,7 @@
 from qibolab.platform import Platform
 from qibolab.pulses import PulseSequence
 from qibolab.qubits import QubitId
+from qibolab.sweeper import Parameter, Sweeper
```

Comment:
```suggestion
from qibolab.sweeper import Parameter, Sweeper, SweeperType
```
We should define the sweepers with explicit type for more clarity

Reply:
Yeah, but I am using the default value, right ?